### ISSUE TITLE
I noticed Exultant is adding Citadel to PhysicalShop

### ISSUE THREAD

**Issue by sirisian** [Exultant/Citadel#70]
Has there been discussion about this for a while? I was hoping you could change the shop system so it would only function if the group on the chest matched the group on the sign. This would also allow public and private citadel options. So if you set the sign to public anyone can use it, but if it's private only people within the group can use it to trade. Just hoping this has been considered.

Just wanted to mention this while people are changing the mod. (The idea that you can break a sign and put up your own with no restrictions has bothered me for a while).

That said I'm not sure what you would put on the last line of the sign anymore. It wouldn't really need the user or group anymore and the format can be changed to:
Buy/Sell
< Amount > < Name >
For
< Amount > < Name >
(Some names might need to be shortened, but it would make the system readable).

Thanks for reading. Not sure if Exultant read the Citadel issues. The PhysicalShop doesn't have issues enabled I don't believe.

**Comment by Exultant** [comment]
The bottom line indicates who owns/created the store.
It might make more sense to have the reinforcement of the sign indicate who is allowed to access the store while the reinforcement on the chest indicate who is allowed to create the store in the first place.

**Comment by sirisian** [comment]
Yeah that would make sense. Also from my previous comment. If you remove the last line just get rid of the Buy/Sell and call it "Trade" since Buy and Sell are redundant. Not sure how much you want to fork from the original PhysicalShop with this. This would make the visual format very easy to read:
Trade
1 Iron
For
64 Experience

As an example and:
Donate
1 Blaze Rod

This would make the whole system look very clean visually.
